oidc-controller/api/authSessions/models.py

```diff
@@ -28,6 +28,7 @@ class Config:
 
 class AuthSession(AuthSessionBase, UUIDModel):
     proof_status: AuthSessionState = Field(default=AuthSessionState.NOT_STARTED)
+    proof_expired_time: Union[datetime, None] = Field(default=None)
```

Comment:
You could likely reuse `expired_timestamp` (even renaming it) in `AuthSessionBase`: I believe it was introduced for this purpose, but never wired-up (I might be wrong, so double checking is best).

Reply:
Same here. Didn't know what `expired_timestamp` was for.

Reply:
I'd use that instead of adding a new property. It should be fairly quick to check whether it is being used somewhere or just set and never used afterwards.